### DISCUSSION TITLE
fail to use pkg install, how can i solve this problem

### DISCUSSION THREAD

**Discussion by pumpkins429** [termux/termux-app#3328] [Q&A] [answered]
i try to use command 'pkg install update' and then it came up with this error mesage 
`W: GPG error: https://mirrors.hit.edu.cn/termux/apt/termux-main stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 5A897D96E57CF20C`

**Comment by sylirre** [comment, marked as answer]
What's your Termux and Android versions?

It looks like either you added a repo in Termux on Android 5 or your environment is messed up.

***

Reinstall Termux app from F-Droid.

If your Android version is 6 or lower, read https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6
